components/integrations/installed_incoming_webhooks/installed_incoming_webhooks.tsx

```diff
@@ -79,6 +82,14 @@ export default class InstalledIncomingWebhooks extends React.PureComponent<Props
         return displayNameA.localeCompare(displayNameB);
     }
 
+    nextPage = () => {
```

Comment:
do we need to update the state on every click?

Reply:
The state of IntsalledIncomingWebhooks, I don't think so? I just need to modify the page value - it can be done outside of state maybe (global variable)?